packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -51,13 +53,18 @@ const modalSizeToClassName: Record<ModalSize, string> = {
 };
 
 export function Modal({
+  defaultTab,
   className,
   overlayClassName,
   children,
   kind = ModalKind.FlexibleCenter,
   size = ModalSize.Medium,
   onRequestClose,
+  tabs,
 }: ModalProps): ReactElement {
+  const [activeTab, onTabChange] = useState<string | undefined>(
```

Comment:
we are making the tabs first class citizen of the modals, therefor no need for any workarounds

Reply:
Like this approach that we have the option to overwrite if need be